src/resolvers.js

```diff
@@ -11,13 +12,33 @@ const resolvers = {
     unixTimestamp() {
       return Math.round(+new Date() / 1000);
     },
+
+    user(obj, args, { token }) {
+      if (!token) { return null; }
+      return {
+        id: 111111,
```

Comment:
OK for MVP? Or is this pre-MVP?

Reply:
I will fix it by implementing token on both sides. and only then we should considr merging.